Directory.Build.props

```diff
@@ -24,10 +24,10 @@
     <!-- The windows SDK version of UWP, Modern UWP and WinUI could be aligned -->
     <UwpMinimum>uap10.0.16299</UwpMinimum>
     <ModernUwpMinimum>net9.0-windows10.0.17763.0</ModernUwpMinimum>
-    <WinUiMinimum>net6.0-windows10.0.18362.0</WinUiMinimum>
```

Comment:
@Sergio0694 I think you said we could align UWP and modern UWP ending part of the version. Do you know if this can be also aligned with winui?

Reply:
> meaning once .NET 8 goes out of support, then both UWP and WinUI could then target the same exact TFM here.

Eh, that might require more infra changes on our side if we will use the same TFM. Our build output simply follows Arcade structure so it's `artifacts/bin/<ProjectName>/<Configuration>/<Tfm>`. We will want to have the output of uwp and winui builds in different directories. But we can think of that later when it will be time to use the same TFM for both.